src/dialog.jsx

```diff
@@ -265,18 +281,18 @@ let Dialog = React.createClass({
     return this.state.open;
   },
 
-  dismiss() {
-    CssEvent.onTransitionEnd(ReactDOM.findDOMNode(this), () => {
-      this.refs.dialogOverlay.allowScrolling();
-    });
+  _testDeprecations() {
+    warning(!this.props.hasOwnProperty('openImmediately'),
```

Comment:
Yeah, I know, but still, uglify is not smart enough to remove code from this method.